XenAdmin/Dialogs/OptionsDialog.cs

```diff
@@ -88,6 +88,7 @@ public static void Log()
             if (!Helpers.CommonCriteriaCertificationRelease)
                 UpdatesOptionsPage.Log();
             DisplayOptionsPage.Log();
+            DisplayOptions2Page.Log();

```

Comment:
You're complete right!

But in my tests the options-dialog displayed not the complete resized page, just the original part. So it was my best option to duplicate it. Can you guide me, how to resize the page properly?

Reply:
I think this is because the layout was chosen in such a way as to avoid scrollbars which do not look very good. You can keep the original size if you compact the controls as I'm suggesting on the screenshots I attached (hold on a bit though before you make any further changes on this, because my screenshots are just suggestions and have not been agreed upon yet).